app/presenters/tree_builder_utilization.rb

```diff
@@ -5,30 +5,26 @@ class TreeBuilderUtilization < TreeBuilder
   has_kids_for EmsFolder, %i(x_get_tree_folder_kids type)
   has_kids_for EmsCluster, [:x_get_tree_cluster_kids]
 
+  def initialize(*args)
+    instance_eval { undef :root_options } unless MiqEnterprise.my_enterprise.is_enterprise?
```

Comment:
@gtanzillo What is the semantics of `is_enterprise?` Do you recall? Is this actually used somewhere in the backend?  Can we currently have more than one `MiqRegion` ?

Thanks.

Reply:
@mzazrivec Wow, `is_enterprise?` was created way back in 2010! The best that I can recall, it's purpose was to detect if the environment is a multi-region environment. I searched the code and it looks the tree builder is the only caller. It's not being used in the backend.

Reply:
@gtanzillo does this mean that its value is permanent? If so, what it is? Can I delete this crazy logic?

Reply:
@skateman My suggestion would be to treat every environment as a mult-region environment and just drop this logic and the `is_enterprise?` method altogether. A single region can be rendered the same way as multiple regions this way.